strongbox-web-core/src/test/java/org/carlspring/strongbox/controllers/layout/maven/MavenArtifactControllerTest.java

```diff
@@ -80,6 +81,8 @@
 
     private static final String TEST_RESOURCES = "target/test-resources";
 
+    private static final String REPOSITORY_RELEASES = "releases";
```

Comment:
Please, make sure that all the repositories required by your test are created by the test itself. The `releases` repository is an existing one and, this way, at any given point, another test could create some artifacts under it, or even purge, or remove the repository (especially, if it's a parallel task). Also, there is a naming convention for these repositories, which is typically, either the initials of the test class' name, or the initials of the test method being prefixed to the repository name. In this case, the test is called `MavenArtifactControllerTest`, hence the prefix would be `mact`. (However, this test used to be called just `ArtifactControllerTest`, before we started having more layout providers and after one of the refactorings, it seems the prefix wasn't changed accordingly).